kakarot_scripts/deployment/starknet_deployments.py

```diff
@@ -34,36 +34,36 @@ async def deploy_starknet_contracts(account):
     class_hash = get_declarations()
     starknet_deployments = get_deployments()
 
-    if NETWORK["type"] is not NetworkType.PROD:
-        starknet_deployments["EVM"] = await deploy_starknet(
-            "EVM",
-            account.address,
-            ETH_TOKEN_ADDRESS,
-            class_hash["account_contract"],
-            class_hash["uninitialized_account"],
-            class_hash["Cairo1Helpers"],
-            BLOCK_GAS_LIMIT,
-        )
-        await invoke(
-            "EVM",
-            "set_coinbase",
-            COINBASE,
-            address=starknet_deployments["EVM"],
-        )
-        starknet_deployments["Counter"] = await deploy_starknet("Counter")
-        starknet_deployments["MockPragmaOracle"] = await deploy_starknet(
-            "MockPragmaOracle"
-        )
-        starknet_deployments["MockPragmaSummaryStats"] = await deploy_starknet(
-            "MockPragmaSummaryStats",
-            starknet_deployments["MockPragmaOracle"],
-        )
-        starknet_deployments["UniversalLibraryCaller"] = await deploy_starknet(
-            "UniversalLibraryCaller"
-        )
-        starknet_deployments["BenchmarkCairoCalls"] = await deploy_starknet(
-            "BenchmarkCairoCalls"
-        )
+    if NETWORK["type"] is NetworkType.PROD:
```

Comment:
This should not appear here 
It seems there is a problem with your rebase

Reply:
I merged instead of rebasing - probably that. I checked the file & compared with the one on main, it looks correct